src/ssl.c

```diff
@@ -6739,9 +6739,11 @@ static int ProcessBufferTryDecodeEd25519(WOLFSSL_CTX* ctx, WOLFSSL* ssl,
 
                 *keyFormat = ED25519k;
                 if (ssl != NULL) {
+#if !defined(WOLFSSL_NO_CLIENT_AUTH) && !defined(NO_ED25519_CLIENT_AUTH)
```

Comment:
There are more checks in internal.h for this field.

Reply:
Agreed, but it will always be there if ED25519 is on and these two macros are not set.

Reply:
If Ed448 is built and not Ed25519 then defining NO_ED448_CLIENT_AUTH will compile out the flag.

Reply:
This function is already wrapped with HAVE_ED25519

Reply:
Right! Thanks!